docs/tuning_fledge.rst

```diff
@@ -126,6 +181,56 @@ In a similar way to the south services, north services and tasks also have advan
 
   - *Data block size* - This defines the number of readings that will be sent to the north plugin for each call to the *plugin_send* entry point. This allows the performance of the north data pipeline to be adjusted, with larger blocks sizes increasing the performance, by reducing overhead, but at the cost of requiring more memory in the north service or task to buffer the data as it flows through the pipeline. Setting this value too high may cause issues for certain of the north plugins that have limitations on the number of messages they can handle within a single block.
 
+  - *Performance Counters* - This option allows for collection of performance counters that can be use to help tune the north service.
+
+Performance Counters
+--------------------
+
+A number of performance counters can be collected in the north service to help characterise the performance of the service. This is intended to provide input into the tuning of the service and the collection of these counters should not be left on during production use of the service.
+
+Performance counters are collected in the service and a report is written once per minute to the storage layer for later retrieval. The values written are
+
+  - The minimum value of the counter observed within the current minute
+
+  - The maximum value of the counter observed within the current minute
+
+  - The average value of the counter observed within the current minute
+
+  - The number of samples of the counter collected within the current minute
+
+In the current release the performance counters can only be retrieved by director access to the configuration and statistics database, they are stored in the *monitors* table. Future releases will include tools for the retrieval and analysis of these performance counters.
```

Comment:
This is a repetitive stuff as same as in section of `South Service Advanced Configuration-> Performance Counters`. We may add this it as  label with reference link. (Likewise Summary of Performance Counter)

Reply:
It was doen deliberately as soem users may just jump to the section that is of interest to them.